container/client.go

```diff
@@ -110,92 +115,93 @@ func (client dockerClient) StopContainer(c Container, timeout time.Duration) err
 }
 
 func (client dockerClient) StartContainer(c Container) error {
+	bg := context.Background();
```

Comment:
Let's not put some semicolon. It's one of my favourite feature of this language :stuck_out_tongue: